portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,31 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments;
```

Comment:
Avoid using implementation types.
https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred

Reply:
Okay, thanks!